tools/validate/src/dependencies/config.ts

```diff
@@ -30,8 +30,15 @@ export const ignoreFilePatterns: Array<RegExp> = [
   /.*\/dist\/.*/,
 ];
 
-// these dependencies will be ignored when listed in a package.json
-export const ignoreMatches = [
+/** If a dependency is flagged as bing imported by one of these files, ignore it */
+export const ignoreFilePatterns: Array<RegExp> = [
+  /.*package.json?/,
+  /.*README.md/,
+  /.*CHANGELOG.md/,
+];
+
+/** these dependencies will be ignored when listed in a package.json */
```

Comment:
how come?

Reply:
These are globally available dev dependencies. We don't want every package flagged for not having React or Storybook or Jest explicitly declared in its package.json

Reply:
Leave that in the comment :)